js/main.js

```diff
@@ -48,36 +35,58 @@ const PHOTOS = [
   'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
 ];
 
-const SIMILAR_ANNOUNCEMENT_COUNT = 10;
+const getRandomNumber = (min = 0, max = 0) => Math.random() * (max - min) + min;
+
+const getRandomInt = (min = 0, max = 0) => Math.floor(getRandomNumber(min, max + 1));
+
+const getRandomDecimalPlace = (min = 0, max = 0, decimalPlaceNumber = 1) => Number(getRandomNumber(min, max).toFixed(decimalPlaceNumber));
+
+const getRandomElement = function (array) {
+  return array[getRandomInt(0, array.length - 1)];
+};
 
-function generateAnnouncement (index) {
-  const announcement = {
+const getRandomElements = function (array = [], count = 0) {
+  const randomElements = new Set();
+  if (count || array.length === 0) {
```

Comment:
Наверное `&&` и `!==`, ведь нам надо собирать массив И если есть ИЗ чего его формировать и если известно ДО какой длины?
```suggestion
  if (count && array.length !== 0) {
```